consumer.go

```diff
@@ -241,20 +241,36 @@ func (c *Consumer) handleRecord(shardID string, r *Record, fn func(*Record) Scan
 }
 
 func (c *Consumer) getShardIDs(streamName string) ([]string, error) {
```

Comment:
hmm looks like my comment go deleted. 

how do you feel about using the built-in pagination functionality:

```go
func (c *Consumer) getShardIDs(streamName string) ([]string, error) {
	var ss []string

	err := c.client.DescribeStreamPages(
		&kinesis.DescribeStreamInput{
			StreamName: aws.String(streamName),
		},
		func(page *kinesis.DescribeStreamOutput, lastPage bool) bool {
			for _, shard := range page.StreamDescription.Shards {
				ss = append(ss, *shard.ShardId)
			}

			return !lastPage
		})

	return ss, err
}
```